### DISCUSSION TITLE
Harmony element root in mscx file is a number but why?

### DISCUSSION THREAD

**Discussion by PogiAbel** [w3c-cg/musicxml#475] [Q&A] [answered]
When I open an mscx fiel as an xml, the harmony element looks like this:
` <Harmony>
 <root>18</root>
  <rootCase>1</rootCase>
   </Harmony>`

However in the [documentation](https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/harmony/) it should be a pitch name and not a number. Does someone know how this number represent the pitch, because I can't figure it out.

**Comment by dspreadbury** [comment, marked as answer]
.mscx is the MuseScore file format, not MusicXML. You will need to consult whatever documentation MuseScore provides for its file format, but I don’t believe it has anything in common with MusicXML (other than being XML-based).

**Reply by PogiAbel** [reply]
Thank you, I will try to not save it as an mscx file rather export to musicxml.
